test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -1016,91 +1067,114 @@ func referenceServerless(name, region, namespace, projectName string, labels map
-	cluster.Spec.AdvancedDeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].AutoScaling = nil
-	cluster.Spec.AdvancedDeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].BackingProviderName = string(provider.ProviderAWS)
-	cluster.Spec.AdvancedDeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].ProviderName = string(provider.ProviderTenant)
-
-	cluster.Spec.AdvancedDeploymentSpec.PitEnabled = pointer.Get(false)
-	cluster.Spec.BackupScheduleRef = common.ResourceRefNamespaced{}
+	cluster.Spec.DeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].ReadOnlySpecs = nil
+	cluster.Spec.DeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].AnalyticsSpecs = nil
+	cluster.Spec.DeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].AutoScaling = nil
+	cluster.Spec.DeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].BackingProviderName = string(akov2provider.ProviderAWS)
+	cluster.Spec.DeploymentSpec.ReplicationSpecs[0].RegionConfigs[0].ProviderName = string(akov2provider.ProviderTenant)
+
+	cluster.Spec.DeploymentSpec.PitEnabled = pointer.Get(false)
+	cluster.Spec.BackupScheduleRef = akov2common.ResourceRefNamespaced{}
 	return cluster
 }
 
-func defaultMaintenanceWindowAlertConfigs() []atlasV1.AlertConfiguration {
-	ownerNotifications := []atlasV1.Notification{
-		{
-			EmailEnabled: pointer.Get(true),
-			IntervalMin:  60,
-			DelayMin:     pointer.Get(0),
-			SMSEnabled:   pointer.Get(false),
-			TypeName:     "GROUP",
-			Roles:        []string{"GROUP_OWNER"},
-		},
+func defaultMaintenanceWindowAlertConfigs(project *akov2.AtlasProject, dictionary map[string]string) []akov2.AlertConfiguration {
+	ownerNotifications := func(i int) []akov2.Notification {
+		return []akov2.Notification{
+			{
+				EmailEnabled: pointer.Get(true),
+				IntervalMin:  60,
+				DelayMin:     pointer.Get(0),
+				SMSEnabled:   pointer.Get(false),
+				TypeName:     "GROUP",
+				Roles:        []string{"GROUP_OWNER"},
+				APITokenRef: akov2common.ResourceRefNamespaced{
+					Name:      resources.NormalizeAtlasName(fmt.Sprintf("%s-api-token-%d", project.Name, i), dictionary),
+					Namespace: project.Namespace,
+				},
+				DatadogAPIKeyRef: akov2common.ResourceRefNamespaced{
+					Name:      resources.NormalizeAtlasName(fmt.Sprintf("%s-datadog-api-key-%d", project.Name, i), dictionary),
+					Namespace: project.Namespace,
+				},
+				OpsGenieAPIKeyRef: akov2common.ResourceRefNamespaced{
+					Name:      resources.NormalizeAtlasName(fmt.Sprintf("%s-ops-genie-api-key-%d", project.Name, i), dictionary),
+					Namespace: project.Namespace,
+				},
+				ServiceKeyRef: akov2common.ResourceRefNamespaced{
+					Name:      resources.NormalizeAtlasName(fmt.Sprintf("%s-service-key-%d", project.Name, i), dictionary),
+					Namespace: project.Namespace,
+				},
+				VictorOpsSecretRef: akov2common.ResourceRefNamespaced{
+					Name:      resources.NormalizeAtlasName(fmt.Sprintf("%s-victor-ops-credentials-%d", project.Name, i), dictionary),
```

Comment:
note to myself: check if this is still executed, this should use generated names.